terraform/kubernetes.tf

```diff
@@ -9,6 +9,12 @@ resource "scaleway_k8s_cluster" "kube_cluster" {
       maintenance_window_start_hour = lookup(var.k8s_auto_upgrade_maintenance_window_start_hour, terraform.workspace, 3)
       maintenance_window_day = lookup(var.k8s_auto_upgrade_maintenance_window_day, terraform.workspace, "any")
    }
+
+   autoscaler_config {
+      scale_down_delay_after_add = lookup(var.k8s_autoscaler_scale_down_delay_after_add, terraform.workspace, "10m")
+      scale_down_unneeded_time = lookup(var.k8s_autoscaler_scale_down_unneeded_time, terraform.workspace, "10m")
+      max_graceful_termination_sec = lookup(var.k8s_autoscaler_max_graceful_termination_sec, terraform.workspace, 2147483647)
```

Comment:
define defaults in terraform/variables.tf? 
Is there a way to make it DRY with what is defined in k8s/base/jvb-deployment.yml?

Reply:
We chose this syntax to be consistent with what Matthieu wrote a few lines above. In my opinion, it would be better to change the whole syntax of the variables used in terraform if it is better to have the default values in the variables.tf file.

It is perhaps possible with a `env.d/common` file used by both Kustomize and Terraform, but it would require some extra development as these tools were developed to be independent in this repository.